debugview/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    alias libs.plugins.android.library
+    alias libs.plugins.kotlin.android
+}
+
+// TODO Uncomment apply plugin once debugview package is ready to be published.
+//if (!project.getProperties()["ANDROID_VARIANT_TO_PUBLISH"].contains("customEntitlementComputation")) {
+//    apply plugin: "com.vanniktech.maven.publish"
+//}
```

Comment:
I'm commenting this out for now until I completely finish the debug view and is ready to be published as a separate package.